plugins/aws/s3/src/hopeit/aws/s3/object_storage.py

```diff
@@ -199,7 +200,10 @@ async def connect(
                 try:
                     await object_storage.create_bucket(Bucket=self.bucket, **kwargs)
```

Comment:
Creating the bucket should be an option of the plugin since it requires different permissions than just writing to an existing bucket. Bucket names are global in AWS, so usually companies require a special procedure to add a new bucket (i.e. terraform). Ok for this PR, but please add a ticket to address this.

Reply:
The creation of a bucket is optional and is defined in the configuration, but cannot be executed programmatically. I have created #6 to address this and all the comments in the review